aws/templates/id/id_sqs.ftl

```diff
@@ -50,13 +29,35 @@
 [#function getSQSState occurrence]
     [#local core = occurrence.Core]
 
-    [#local id = formatComponentSQSId(core.Tier, core.Component, occurrence)]
+    [#local id =
+        formatComponentResourceId(
+            SQS_RESOURCE_TYPE,
+            occurrence.Core.Tier,
+            occurrence.Core.Component,
+            occurrence) ]
+    [#local name = (core.Component.Name != "SQS")?then(
+                            formatName(
+                                core.Component.Name,
+                                occurrence),
+                            formatName(
+                                productName,
+                                segmentName,
+                                componentName,
+                                occurrence))]
+
+    [#local dlqId = formatDependentResourceId(SQS_RESOURCE_TYPE, id, "dlq") ]
```

Comment:
Added the resource type here would make it sqsXsqs wouldn't it?

Reply:
yes.

The first sqs is the resource type, the next is the type of the resource it depends on - in this case an SQS queue as well.